plutus-core/plutus-ir/test/PlutusIR/Parser/Tests.hs

```diff
@@ -79,23 +79,46 @@ aroundSeparators = go False False
                 pure $ a : s1 ++ b : s2 ++ rest
             | otherwise -> (a :) <$> go inQuotedName inUnique splice (b : l)
 
+-- | Check whether the given constant can be scrambled (in the sense of 'genScrambledWith').
+isScramblable :: PLC.Some (PLC.ValueOf PLC.DefaultUni) -> Bool
+isScramblable (PLC.Some (PLC.ValueOf uni0 x0)) = go uni0 x0 where
```

Comment:
I hate the verbosity and awkwardness of this definition and the `isPrintable` one, but I'm going to refrain from spending time on trying to make it nicer. Ultimately, we should split the universe into the type-level and term-level ones and this will make this definition quite nicer.